#!/usr/bin/env python

PACKAGE = "vn_100"

import roslib;roslib.load_manifest(PACKAGE)

#form dynamic_reconfigure.parameter_generator import *

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()


gen.add("kp_yaw" , double_t , 0 , "kp_yaw value",0,0,2)
gen.add("ki_yaw" , double_t , 0 , "ki_yaw value",0,0,2)
gen.add("kd_yaw" , double_t , 0 , "kd_yaw value",0,0,2)
gen.add("setpoint_yaw" , double_t , 0 , "setpoint_yaw",0,-30,30)
gen.add("kp_depth" , double_t , 0 , "kp_depth value",232,0,500)
gen.add("ki_depth" , double_t , 0 , "ki_depth value",0,0,100)
gen.add("kd_depth" , double_t , 0 , "kd_depth value",0,0,100)
gen.add("setpoint_depth" , double_t , 0 , "setpoint_deph",996,992,1300)
gen.add("bool_param",bool_t,0,"a booleam parameter" , False)



exit(gen.generate(PACKAGE, "vn_100" , "pid"))


